lib/BrizyForms/Service/WebHooksService.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace BrizyForms\Service;
+
+use BrizyForms\Exception\ServiceException;
+use BrizyForms\FieldMap;
+use BrizyForms\Model\Account;
+use BrizyForms\Model\Folder;
+use BrizyForms\Model\Group;
+use BrizyForms\Model\GroupData;
+use BrizyForms\Model\RedirectResponse;
+use BrizyForms\Model\Response;
+use BrizyForms\ServiceConstant;
+use GuzzleHttp\Client;
+class WebHooksService extends Service
+{
+
+    private static function curlExec($value)
+    {
+        $client = new Client();
+
+        $options = array();
+        if(array_key_exists('header', $value))
+        {
+            $options['headers'] = $value['header'] ;
+        }
+
+        if(array_key_exists('message', $value)) {
+            $options['form_params'] = $value['message'];
+        }
+
+        $client->request($value['sendType'], $value['url'], $options);
```

Comment:
if the request is invalid throw the [ServiceExcecption](https://github.com/bagrinsergiu/Brizy-Forms-SDK/tree/master/lib/BrizyForms/Exception)